sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze.py

```diff
@@ -477,382 +277,185 @@ def test_out_of_order_ids_multiple_tasks(self, client):
                 {"id": "19", "text": ":P"},
                 {"id": "1", "text": ":D"}]
 
-        response = client.begin_analyze(
+        response = client.begin_analyze_batch_actions(
             docs,
-            entities_recognition_tasks=[EntitiesRecognitionTask(model_version="bad")],  # at this moment this should cause all documents to be errors, which isn't correct behavior but I'm using it here to test document ordering with errors.  :)
-            key_phrase_extraction_tasks=[KeyPhraseExtractionTask()],
-            pii_entities_recognition_tasks=[PiiEntitiesRecognitionTask()],
+            actions=[
+                RecognizeEntitiesAction(model_version="bad"),
+                ExtractKeyPhrasesAction(),
+                RecognizePiiEntitiesAction(),
+            ],
             polling_interval=self._interval(),
         ).result()
 
-        results_pages = list(response)
-        self.assertEqual(len(results_pages), 1)
-
-        task_types = [
-            "entities_recognition_results",
-            "key_phrase_extraction_results",
-            "pii_entities_recognition_results"
-        ]
+        action_results = list(response)
+        assert len(action_results) == 3
 
-        in_order = ["56", "0", "19", "1"]
+        assert action_results[0].action_type == AnalyzeBatchActionsType.RECOGNIZE_ENTITIES
```

Comment:
sorry, I am lost and it could be because of Python itself :( 
so how does the user know that the `RECOGNIZE_ENTITIES` action failed? where do u surface the error?

Reply:
users will do the `is_error` check. I'll make sure to update the readme and samples so this is clear. thanks!